lib/car.rb

```diff
@@ -0,0 +1,31 @@
+class Car
+
+  attr_reader :make_model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make_model = make_model
```

Comment:
Since we don't really need access to `make_model` later, we could initialize @make and @model here:

```
make_model = make_model
@make = make_model.split()[0]
@model = make_model.split()[1]
```

just another option :)